data_model/src/transaction.rs

```diff
@@ -165,7 +165,7 @@ model! {
     ///
     /// Uses **base64** (de-)serialization format.
     #[derive(DebugCustom, Clone, PartialEq, Eq, Hash, Constructor, Decode, Encode, Deserialize, Serialize, IntoSchema)]
-    #[debug(fmt = "WASM binary(len = {})", self.0.len())]
+    #[debug(fmt = "WASM binary(len = {})", "self.0.len()")]
```

Comment:
@Arjentix I believe this is how it should be done

Reply:
Without it it was `[0, 1, 2, 3]` etc with all the bytes inside.